sigopt/interface.py

```diff
@@ -73,6 +73,14 @@ def __init__(self, requestor, api_url=None, user_agent=None, verify_ssl_certs=No
       ],
     )
 
+    best_training_runs = ApiResource(
```

Comment:
fwiw i think this change is in the main branch (PR #214) - only difference I see is that i used `object_or_paginated_objects` (and I don't know the difference, so this may be more correct, i was just copying i think best-observations endpoint)

Reply:
`object_or_paginated_objects` is used for example here:
`GET /v1/experiments`
`GET /v1/experiments/X`
where the associated python calls are
`conn.experiments().fetch()` and `conn.experiments.fetch(X)`

in this case the python implementation does not differentiate the return types between the two, hence `object_or_paginated_objects`.

In this case `GET /v1/experiments/X/best_training_runs/Y` is not a valid endpoint so we just need `paginated_objects`

Reply:
ah ok cool, thanks. that may trigger a merge conflict then, feel free to handle that whenever nd keep it the way you have.

Reply:
Yeah I noticed the conflict